internal/integration/rest/models.go

```diff
@@ -63,6 +63,26 @@ type (
 		ParendID int    `json:"parentId"`
 	}
 
+	Project struct {
+		ID                 int                   `json:"id"`
+		TeamID             int                   `json:"teamId"`
+		Name               string                `json:"name"`
+		IsPublic           bool                  `json:"isPublic"`
+		SourceSettingsLink SourceSettingsLinkObj `json:"sourceSettingsLink"`
```

Comment:
From [API docs](https://checkmarx.atlassian.net/wiki/spaces/SAST/pages/3206357767/CxSAST+REST+API+v1) it seems this is an API link to another resource, and doesn't belong to the export.
Instead, we could use the link to find the sourceSettings object, and then include it in the export.

Reply:
Do you mean to take link from here?
`"link": {
      "rel": "string",
      "uri": "string"
    }`
From uri field